src/config/dark-sites.config

```diff
@@ -207,6 +207,7 @@ devanbuggay.com
 devart.withgoogle.com
 developerinsider.co
 developpement-systeme-exploitation.github.io/documentation
+dev.minefly.org
```

Comment:
This also doesn't seem to add up?
![image](https://user-images.githubusercontent.com/25481501/135896321-37ba94fa-8c90-4b32-9667-144b42ca1741.png)